utils/lib/parse_usage.py

```diff
@@ -0,0 +1,20 @@
+import re
```

Comment:
This is all based on a VPR specific format, right? If so maybe explain that in a docstring.

Reply:
I updated the docstring with more detail, take a look.